README.md

```diff
@@ -74,5 +74,7 @@ Refer [hello_world.ipynb](https://github.com/mozilla/overscripted/blob/master/an
 
 - [This](https://github.com/aSquare14/Git-Cheat-Sheet) will help you get started with GIT. For visual thinkers this [tutorial](https://www.youtube.com/playlist?list=PL6gx4Cwl9DGAKWClAD_iKpNC0bGHxGhcx) can be a good start.
 
+- Or refer this quick Dask [overview](https://www.youtube.com/watch?v=ods97a5Pzw0) video or the Dask [Cheatsheet](https://dask.pydata.org/en/latest/_downloads/daskcheatsheet.pdf) to get familiar with the workflow.
```

Comment:
This renders as: "Or refer this quick Dask overview video or the Dask Cheatsheet to get familiar with the workflow."

The "Or" at the start of this sentence does not relate to the previous bullet point. Suggest adding to the existing bullet on dask.

We can clean up the english and capitalization and keep it simple. "Other Dask resources: overview video, cheatsheet." (obviously with the links added).